cookbooks/controller_conf/attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['controller_conf']['host_ip'] = ""
 default['controller_conf']['service_host'] = ""
+default['devstack_conf']['git_base'] = "https://github.com"
```

Comment:
I'm not confident on my comment, though, does 'devstack_conf' come from other recipe? why is it not 'controller_conf' rather than 'devstack_conf'?